runtimes/crystal-1.7.1/README.md

```diff
@@ -0,0 +1,111 @@
+# Crystal Runtime 1.7.1
+
+This is the Open Runtime that builds and runs Crystal code based on a `crystal:1.7.1-alpine` base image.
+
+The runtime itself uses [Grip](https://github.com/grip-framework/grip) as the web server to process the execution requests.
+
+To learn more about runtimes, visit [Structure](https://github.com/open-runtimes/open-runtimes#structure) section of the main README.
+
+## Usage
+
+1. Create a new Crystal project and enter the project directory.
+
+2. Rename the generated file in `/src` to `main.cr`.
+
+3. Add the following code to `main.cr`:
```

Comment:
Not for myself to discuss this with Devrel ( @gewenyu99 )

I can see how `echo` script would make this really unreadable. But consistency is still a key. We should find a good way to provide this script that is readable and copyable. Then we should apply such scheme to all readmes.

Ill keep you posted about our decision.